### ISSUE TITLE
Now that I have the annotated csv file, how do I get the h5 file I need to create the training dataset?

### ISSUE THREAD

**Issue by Zch6666** [DeepLabCut/DeepLabCut#2254]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Zch6666** [comment]
Now that I have the annotated csv file, how do I get the h5 file I need to create the training dataset?

**Comment by MMathisLab** [comment]
The H5 is created automatically when you label with DeepLabCut; the CSV is never used in our codebase, it's just for ease of human readability. Please also follow the template rules (you can edit posts on github), and in general we ask github stay for code issues and questions go to ImageForum 🤗 https://github.com/DeepLabCut/DeepLabCut#get-assistance--be-part-of-the-dlc-community

**Comment by Zch6666** [comment]
But in the paper, "A 3D virtual mouse generates synthetic training data for synthetic mice
behavioral analysis doi:10.1038/s41592-021-01103-9. He created a training dataset by converting a csv file from Blender into an H5 file.

**Comment by MMathisLab** [comment]
That paper isn't related to our project. 

There are tons of ways to convert csv to h5; if you Google this you'll find many hits on stackoverflow, etc... 

https://stackoverflow.com/questions/27203161/convert-large-csv-to-hdf5

**Comment by Zch6666** [comment]
Thank you for your reply！
